conf/version.go

```diff
@@ -2,4 +2,4 @@ package conf
 
 // VERSION is the app-global version string, which should be substituted with a
 // real value during build.  Follows semantic versioning: MAJOR.MINOR.PATCH
-var VERSION = "v0.12.6"
+var VERSION = "v0.1.1"
```

Comment:
Do we need to bump if no functional change to the binary?

Reply:
Wonder if `VERSION` could set via a file that `release.sh` auto increments, say `version.txt` that just has `v0.1.0`.  `release.sh` could take `--version patch|minor|major` to determine how to increment.

Thinking out-loud.  If we need more automation, we could revisit.

Reply:
I wanted to keep things simple for now.

But we certainly could do more sophisticated stuff in the future. It would make the action more complicated, since the tag has to be on the main branch and any change to version file on a PR/branch. We'd need to create a PR via action, update the version file, make sure there aren't conflicts, merge PR and then push tags, etc. If there are other in-flight changes we'd have to manage that as well.

One more thing missing here that we should automate in the future is release notes/changelog.

Reply:
I need to test if it actually works for real, so do need to cut a release. :)
That's why I was thinking v0.1.1.

If we don't want to bump the version, I can tag the existing one with a temp v0.0.9 release, delete the v0.1.0 tag from it and re-release v0.1.0 via automation, then delete v0.0.9...

Let me know what you prefer and what the impact of bumping the version will be.

Reply:
Bumping should be fine.